.github/workflows/golangci-lint.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Run go lint
         run: |
           # The following packages are needed to build Fleet Desktop on Ubuntu.
-          sudo apt install -y gcc libgtk-3-dev libayatana-appindicator3-dev
+          sudo apt update && sudo apt install -y gcc libgtk-3-dev libayatana-appindicator3-dev
```

Comment:
should we also pass the `-y` flag to `apt update`? I believe there are certain scenarios where `apt update will prompt.

Reply:
good catch, thanks!